test/parallel/test-https-pfx.js

```diff
@@ -28,7 +28,7 @@ const assert = require('assert');
 const fs = require('fs');
 const https = require('https');
 
-const pfx = fs.readFileSync(`${common.fixturesDir}/test_cert.pfx`);
+const pfx = fs.readFileSync(`${common.fixtures}/test_cert.pfx`);
```

Comment:
It seems like the task description was probably not clear enough. I am not sure if this would actually pass or not but the idea was to require the `../common/fixtures` and to use the `path` function from that module instead of using the more verbose version here.

Reply:
@tkg214 thank you very much for participating in the code-and-learn. As @BridgeAR indicates, there are some issues with the changes. This should be:

```js
const pfx = fixtures.readSync('test_cert.pfx');
```